autosklearn/data/xy_data_manager.py

```diff
@@ -24,7 +24,7 @@ def __init__(
         X_test: Optional[np.ndarray],
```

Comment:
Are these data types still correct?

Reply:
Not anymore, I have updated this class typing.